tests/view_tests/tests/test_defaults.py

```diff
@@ -123,3 +128,22 @@ def test_default_content_type_is_text_html(self):
 
         response = self.client.get('/server_error/')
         self.assertEqual(response['Content-Type'], 'text/html')
+
+    def test_custom_templates_wrong(self):
+        """
+        Default error views should raise TemplateDoesNotExist when passed a
+        template that doesn't exist.
+        """
+        rf = RequestFactory()
+        request = rf.get('/')
+        with self.assertRaises(TemplateDoesNotExist):
+            bad_request(request, Exception(), template_name='...')
```

Comment:
Looks like Windows isn't happen with the dots. I think something like 'nonexistent' should work.